palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/model/PalisadeClientResponse.java

```diff
@@ -26,18 +26,18 @@
 import java.util.StringJoiner;
 
 /**
- * Response message that is returned to the client.  The message contains information that will identify this request
- * for access to the data and be used in a subsequent request to see the resources.
+ * Response message that is returned to the client.
+ * This message contains a unique token to link the request from the client to the data, and is used at the Filtered Resource Service to retrieve data.
```

Comment:
I think Data Service would be the right one to mention in a sentence about the client.  Alternatively re-word to mention both.

Reply:
updated to include a link to the Data Service.